WalletWasabi/Helpers/ErrorDescriptor.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+
+namespace WalletWasabi.Helpers
+{
+	public class ErrorDescriptors : List<ErrorDescriptor>
```

Comment:
1 class = 1 file: https://stackoverflow.com/questions/2434990/one-class-per-file-rule-in-net